index.rst

```diff
@@ -1,49 +1,32 @@
-.. Freedom of the Press Foundation's Redmine documentation master file, created by
-   sphinx-quickstart on Wed Mar  9 12:37:53 2016.
-   You can adapt this file completely to your liking, but it should at least
-   contain the root `toctree` directive.
-
 SecureDrop Support Documentation
 ================================
 
-Overview
---------
-
-Freedom of the Press Foundation is transitioning to a Redmine-based
-ticketing system for all support requests related to SecureDrop. All
-current and future SecureDrop administrators are required to use the new
-support system in order to receive support from FPF staff. From now on,
-the best way to get in touch with us is through our new `support site
-<https://support.freedom.press>`_ or by emailing support@freedom.press.
+Freedom of the Press Foundation uses a `Redmine`_-based ticketing system for all
+support requests related to SecureDrop. This system offers simple, centralized
+assistance to the administrators who maintain SecureDrop instances and the
+journalists who use them. Running a SecureDrop is hard work. We at FPF are
+committed to offering support during installation and into the working life of
+an organization's SecureDrop instance.
 
-As a current SecureDrop administrator or journalist, you will need to
-spend a little time setting up an account on the new support site. This
-is quick and easy - get started by :doc:`creating_your_account`.
+.. note:: The `support site`_ is also available as a Tor Onion Service at
+          http://support6kv2242qx.onion/.
 
-.. note:: We recently added the ability to reach our new support site
-          through a Tor onion service. It's available at
-          http://support6kv2242qx.onion/
-
-First time contacting us?
--------------------------
+.. _`Redmine`: https://www.redmine.org/
+.. _`support site`: https://support.freedom.press/
 
-If you've never contacted us before, and have questions about:
+Getting Started
+---------------
 
-* Installing SecureDrop
-* An existing SecureDrop installation that was set up without our involvement
-* Inclusion in the `SecureDrop Directory <https://securedrop.org/directory>`_
+Current SecureDrop users can get started by :doc:`creating an account
+<creating_your_account>`.  Once your account has been created, there are two
+ways to receive support: a :doc:`web-based system <using_redmine>` (often the
+preferred method), and an :doc:`email-based system
```

Comment:
"System" doesn't strike me as the best term here. I know "workflow" is wonky as well. How about just rewording this completely: "You can receive support via email or through the web", or something similarly simple.